cluster/examples/kubernetes/edgefs/sslKeyCertificate.yaml

```diff
@@ -0,0 +1,9 @@
+  sslcert: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURQRENDQWlRQ0NRRFRoYm9mSzY1MFZ6QU5CZ2txaGtpRzl3MEJBUXNGQURCZ01Rc3dDUVlEVlFRR0V3SlYKVXpFVE1CRUdBMVVFQ0F3S1EyRnNhV1p2Y201cFlURVVNQklHQTFVRUJ3d0xVMkZ1ZEdFZ1EyeGhjbUV4RURBTwpCZ05WQkFvTUIwNWxlR1Z1ZEdFeEZEQVNCZ05WQkFNTUMyNWxlR1Z1ZEdFdVkyOXRNQjRYRFRFMk1EUXdOekF4Ck1Ea3pNbG9YRFRFNU1ERXlOakF4TURrek1sb3dZREVMTUFrR0ExVUVCaE1DVlZNeEV6QVJCZ05WQkFnTUNrTmgKYkdsbWIzSnVhV0V4RkRBU0JnTlZCQWNNQzFOaGJuUmhJRU5zWVhKaE1SQXdEZ1lEVlFRS0RBZE9aWGhsYm5SaApNUlF3RWdZRFZRUUREQXR1WlhobGJuUmhMbU52YlRDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDCkFRb0NnZ0VCQUxmUU9mYkdUMldZQ2dNSHFHS2tRVmVYcldnTmcraHAvYmVBWE9VUWhnY3JOTEFrd2pFNnI3UXAKWHR1dDRpNHVmb2dFTzhUeVpUcnJwaUQ2WU9PSFVrbDVKcy9CN2VmNloxMmZETnk5OG12OTFia2cvYjhtRFRsaApnSU4vWFJRSngxQ1l0eGM2TklJc3dWNGhLZWF3WW02cThEQUkxNDlPWklQNVBqSm85ZkhEbUFqYWFCNkNNQ2VpCkRvdDUrM1BIcDZUNkh5NUFzVGluTHV0ZG95SjRWa21QekZ0NHY5WlJRR2hkOXI1b2xGOGdiVjZ2WmtYTGRtZFkKT3llVnVsVGxjZ0ppSDEzT3hFUU9MaTNBNXBQTDRrSHZxYlhmSHMvTmFOUXRXcDRuUG1OUmtpbmV0cnVXV2RkcAo1cllQSVc2RkEwV2d1cXlyRC9LUmQvaFQxenFvWmVFQ0F3RUFBVEFOQmdrcWhraUc5dzBCQVFzRkFBT0NBUUVBClJOcmhVdkNoY2kxVjFISVZ0dlhMbVFTS1BmV1pRSmlOakxndFoyVHNSUHludUtvc2FqM1BVSWYyTndIZ2dkd3MKLy8rWUFHZHBDV21XZEljRlBEbEoxMVVUSk9sQUlxR1R5N3ZLV29EN2dKMExWdHFTVk5maEdqKzd3MHZ6b0tuYwphUVdaOURVVkFDZXBCUzdMNkQxNFV3TmhhMkdiN0NmaFdEeGM5c1A3VlFoUW1tbUhGcWtzaTRlTGxrN05Zd0ZQCkMwVW9FWWpSZnl3bTl3cjVFVXBQR1hHUW9sMTdtSUpmVEYzejYvUmFWdlBITEY3VVZxS0YvRFgraTlLci9LUWIKS2JpTGpETXJUTGgxOUQvbFArdzJNR3ZVZE5BMDByUG4vMldMdEVQNktVYWFiMGt3L05EUkY5NjBNSjJJN3Q3aApHSk8yNFdWd0l2bjZlaGVkMmdqOWZ3PT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
```

Comment:
is this a self-signed or public cert that you're OK with being published into the repo? just want to make sure

Reply:
it is public and referenced here only as example how to format sslkey and sslcert.